ui/src/app/cohort-search/list-search/list-search.component.tsx

```diff
@@ -450,7 +450,7 @@ export const ListSearch = fp.flow(withCdrVersions(), withCurrentWorkspace(), wit
 
     getConceptLink(conceptId) {
       return environment.publicUiUrl + '/ehr/' +
-        domainToTitle(this.props.searchContext.domain).toLowerCase() + '?search=' + conceptId;
+        domainToTitle(this.props.searchContext.domain).toLowerCase() + '/' + conceptId;
```

Comment:
Works for conditions and procedures, but the names for drugs and measurements don't match with data browser and we get an infinite spinner. This sets 'drugs' and 'labs and measurements' in the link but data browser uses 'drug-exposures' and 'labs-and-measurements' (has probably been like this for a while, but we should fix it since we're changing the link anyway). Not sure the best way to set the correct text, maybe a switch statement?

Reply:
Also noticed even when the search works, I never get results using the concept id. Only keywords seem to return results. 

https://user-images.githubusercontent.com/40036095/127554629-8900ff41-e419-465e-91d9-133305daf73f.mov


@SrushtiGangireddy Do you know if the search should work when passing a concept id in the routes params?

Reply:
@dolbeew The concept ids should usually return the search results there are few top level concepts that don't show up though like 4042140 (Finding by site) since it is hidden from databrowser tool.